tox.ini

```diff
@@ -15,5 +15,5 @@ commands =
     pytest -v {posargs}
 
 [flake8]
-ignore = W503,W605,E128,E501,E203,E266,E501,E231
+ignore = W503,W605,E128,E501,E203,E266,E501,E231,SFS101,SFS201,SFS301
```

Comment:
We don't use the SFS plugin, so these ignore-statements are not necessary

Reply:
They were necessary in my shared sandbox.  I will remove them.